data-dir-server.patch

```diff
@@ -0,0 +1,38 @@
+--- a/src/server/servermain.cpp	2014-03-22 14:31:22.000000000 +0100
++++ b/src/server/servermain.cpp	2020-01-13 20:53:23.473706499 +0100
+@@ -60,7 +60,7 @@
+ static bool g_run_in_foreground = false;
+ static bool g_print_syslog_to_stderr = false;
+ static bool g_workaround_memleaks = false;
+-static std::string g_config_file = "server/server.xml";
++static std::string g_config_file = "server.xml";
+ static std::string g_rules_file = "";
+ 
+ // ...
+@@ -85,7 +85,7 @@
+ 	process_arguments(argc, argv);
+ 
+ 	FileSystem fileSys(argv[0]);
+-	fileSys.addToSearchPath("data" + fileSys.getDirSeparator() + "rules.zip");
++	fileSys.addToSearchPath("/app/share/blobby" + fileSys.getDirSeparator() + "rules.zip");

```

Comment:
Avoid hardcoding the patch & send it as an upstream patch if possible.

Reply:
As I wrote, this kind of patch was accepted by upstream developer in the Flatpak pakcage. He may want to resolve this upstream if he wants but it does not depend on me. IMHO currently it's fine.  Can we keep it at least for the start?

Reply:
I'm not sure I understood what you mean, but at worst an issue could be opened so that upstream is aware of that

Reply:
As I wrote in PR description, the whole Flatpak package definiton was consulted with upstream (please see link in PR description) and upstream maintainer is aware of these patches. If he decide to upstream them, he will do this, but I am not part of development team.